src/graphics/program-lib/chunks/normal.vert

```diff
@@ -6,6 +6,24 @@ vec3 getNormal() {
     #else
         dNormalMatrix = matrix_normal;
     #endif
-    return normalize(dNormalMatrix * vertex_normal);
+
+    vec3 tempNormal = vertex_normal;
```

Comment:
is vertex_normal used elsewhere? shouldn't it be updated directly by morph weights?

Reply:
this seems the only place where it's used in generic sense. There's a specialized chunk for skinning only, and for instancing only.

You cannot modify vertex_normal, I tried that, those are input attributes and you get shader compiler error.

This seems consistent with previous state of this file - we modify vertex_normal by skinning and instancing, I just added morphing.